### DISCUSSION TITLE
How long does it take for a reviewer to be assigned to a pull request?

### DISCUSSION THREAD

**Discussion by dasblinkenlight** [dotnet/aspnetcore#51463] [Q&A] [answered]
Could you please give me a general idea on the number of days it takes for a reviewer to be assigned to a pull request? I am looking for a ballpark figure - are we talking weeks? Months? The "how to contribute" page does not mention anything in terms of the timeline, and I have not been through the process of submitting a PR before, so I would like to get a general idea of the timing.

**Comment by martincostello** [comment, marked as answer]
From personal experience, it can vary from hours to maybe a week or two.

It depends on when you open it (Is it a weekend? Is it during the team's working hours?), how busy the team are, what you've changed, and how big/complicated/risky the change is, etc.

**Reply by dasblinkenlight** [reply]
I see, thank you very much. The bot has requested a reviewer for my PR, but it does not look like that reviewer is real, because its username has a forward slash (dotnet/minimal-apis).

I did open it on a weekend, and the change does look big (>95% of the change are the additional unit tests and the control output).

**Reply by martincostello** [reply]
That will be the name of a GitHub team, whose membership isn't visible to people who aren't members of the dotnet organisation.

**Comment by danmoseley** [comment]
Cc @mkArtakMSFT who is looking into how we can be more responsive on PRs.
